### DISCUSSION TITLE
TP5 - ejercicio 3 - Fracciones

### DISCUSSION THREAD

**Discussion by AnGys09** [INGCOM-UNRN-P1/cursada-2023#167] [Preguntas y respuestas] [answered]
Hola buenas tardes, consulta, en el ejercicio de fracciones tenemos que: 
Operaciones a implementar:
 * Mostrar una fracción por consola.
 * Suma entre fracciones
 * Multiplicación entre fracciones
 * Suma con un número entero.

Mi consulta es: que sentido tiene hacer dos códigos idénticos, si  matemáticamente es lo mismo sumar fracción con fracción o fracción con entero, osea.... tenemos que implementar algo diferente entre una y otra y a mi no se me ocurre, o que  tenemos que hacer  en si para,  no copiar  y usar la misma, Gracias.

**Comment by martinvilu** [comment, marked as answer]
¿Y por qué tienen que ser idénticos?, siendo que uno puede llamar al otro.

**Comment by martinvilu** [comment]
Probablemente, la suma con un número entero se pueda expresar como
```C
fraccion_t argumento = {.numerador = numero, .denominador = 1};
return suma_fraccion(fraccion, argumento);
```
